src/dali/plugin/input/csv/manual.py

```diff
@@ -109,43 +163,51 @@ def _load_in_file(self, transactions: List[AbstractTransaction]) -> None:
         with open(self.__in_csv_file, encoding="utf-8") as csv_file:
             lines = reader(csv_file)
             header_found: bool = False
-            for line in lines:
+            for line_idx, line in enumerate(lines):
                 raw_data: str = ",".join(line).strip()
                 if not header_found:
                     # let user know there is not enough columns
-                    if len(line) - 1 < self.__IN_NOTES_INDEX:
-                        raise ValueError(f"Not enough columns: the {self.__in_csv_file} CSV must contain {self.__IN_NOTES_INDEX} columns.")
+                    ipdb.set_trace()
+                    # I'm not sure I agree with this check. This means every user must use every column header, even optional ones,
+                    # so this will break everyone's existing config files every time a new column optional or not, is added. 
+                    #Why don't we check for the length of required parameters only? We'll have to add IndexError try/catch. See below
```

Comment:
I think this checks only for required parameters (see definition of `__required_columns` above (line 141).